db/db_impl/db_impl.cc

```diff
@@ -2315,6 +2328,7 @@ Status DBImpl::MultiGetImpl(
       super_version->current->MultiGet(read_options, &range, callback,
                                        is_blob_index);
     }
+    curr_value_size = range.GetValueSize();
```

Comment:
Should we bail here if ```value_size``` exceeds that specified by user, instead of continuing to iterate over the batch?